content/en/getting_started/application_security/software_composition_analysis.md

```diff
@@ -31,19 +31,26 @@ Datadog [Software Composition Analysis][1] (SCA) continuously monitors your prod
 This guide walks you through best practices for getting your team up and running with SCA.
 
 ## Phase 1: Enable
-1. Check the ASM Library Compatibility for each ASM product to see if your service is supported.
-2. Enable Software Composition Analysis on your services. 
-   - Navigate to the [Quick Start Guide][4].
-   - Expand **Enable Vulnerability Detection**.
-   - Click **Start Activation**.
-   - Choose services to secure with ASM.
-
-   OR
-   
-   - Navigate to the [Setup][9] page.
-   - To enable Software Composition Analysis for static analysis in source code, in **Software Composition Analysis**, click **Get Started**.
-     - In **SCA static analysis in source code**, click **See Instructions**.
-     - In **SCA runtime analysis in running services**, click **Select Services**.
+
+1. Refer to the [Library Compatibility Requirements][12] page to verify if the Datadog Tracing Library used by your application or service supports the Software Composition Analysis (SCA) capability for your application's or service's programming language.
+2. Enable Software Composition Analysis on your services.
+   1. By the Quick Start Guide:
+      1. In Datadog, go to [Application Security > Settings > Quick Start Guide][4].
+      2. Expand **Enable Vulnerability Detection**, select **Open source vulnerabilities**, and click **Start Activation**. A list of services appears.
+      3. Select the service(s) you want to monitor for vulnerabilities, then click **Next**. The number of selected services and their names are listed.
+      4. Click **Enable for Selected Service(s)** to complete the activation of Software Composition Analysis (SCA) for the chosen service(s).
+      {{< img src="/security/application_security/software_composition_analysis/APM_SCA-enablement-quick-start-guide.mp4" alt="quick start guide in the Datadog UI" video="true">}}
+3. By the Settings page:
```

Comment:
```suggestion
   b. By the Settings page:
```

Reply:
It looks like this is the next item after the `By the Quick Start Guide` option, so updating the indentation to match. Let me know if that's not the case.